6.4.remember_remember.py

```diff
@@ -0,0 +1,26 @@
+from PIL import Image
+
+
+def decoding_an_image(path):
+    # Open the image
+    try:
+        img = Image.open(path)
+    except:
+        print("Wrong path!")
+        return
+    width = img.width
+    height = img.height
+
+    my_string = ''
+    # Goes over all pixels
+    for i in range(0, width):
+        for j in range(0, height):
+            # Check if pixel is black
+            if img.getpixel((i, j)) == 1:
+                # Convert the line number of black pixel to character by ASCII value
+                my_string += chr(j)
+
+    return my_string
+
+
+print(decoding_an_image('code.png'))
```

Comment:
Use _name==main_ - 
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/

Reply:
fixed it